tests/fixtures/common/devfile/devfile.yaml

```diff
@@ -0,0 +1,15 @@
+schemaVersion: 2.2.2
+metadata:
+  name:
```

Comment:
This line is invalid, give it a valid name

Reply:
We will need to construct a name here, something like this: https://github.com/ansible/ansible-creator/blob/9ae5062d74e87c31b62f42b6f859f2d5a6569029/src/ansible_creator/subcommands/init.py#L101

Because we don't have an context about the project type I think we could use the directory name followed by a short random number or something.

Reply:
( you'll probably need to monkeypatch whatever function you create for the tests since it would be different every run)

Reply:
Done!